src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentTwigExtension.php

```diff
@@ -112,6 +113,10 @@ public function load($uuid)
                 $locale
             );
 
+            if (false === $contentStructure instanceof PageInterface) {
```

Comment:
Can you check if something like:

```
$document = $contentStructure->getDocument();

if ($document && 'page' === $document->getStructureType()) {
```

Works here.

I'm afraid that some custom bundles extended from the `PageBridge` and they would run into the same issue if you check for the Interface.